packages/edit-navigation/src/store/test/resolvers.js

```diff
@@ -28,288 +37,163 @@ jest.mock( '@wordpress/blocks', () => {
 	};
 } );
 
-describe( 'getNavigationPostForMenu', () => {
-	it( 'returns early when a menuId is not provided', () => {
-		const generator = getNavigationPostForMenu( null );
-		expect( generator.next().value ).toBeUndefined();
-		expect( generator.next().done ).toBe( true );
-	} );
-
-	it( 'gets navigation post for menu id', () => {
-		const menuId = 123;
+function createRegistryWithStores() {
+	// create a registry and register stores
+	const registry = createRegistry();
+
+	registry.register( editNavigationStore );
+	registry.register( coreStore );
+
+	// Set up the navigation post entity.
+	registry.dispatch( coreStore ).addEntities( [
```

Comment:
This is copied over from `index.js`. I thought about making it a thunk and importing in both places, but it wasn't clear how would I dispatch it here so I just copied that fragment over.